compiler/src/dmd/globals.d

```diff
@@ -222,6 +222,7 @@ extern (C++) struct Param
     const(char)[] resfile;
     const(char)[] exefile;
     const(char)[] mapfile;
+    bool disableLinkerStripDead; // do not try to remove unused symbols during linking
```

Comment:
Dead field, just don't bother adding it.

Reply:
are you asking how to add a CLI flag?